### DISCUSSION TITLE
native Frigate installation HowTo  (w/o Docker)

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#4030] [General]
Hello forum

Is there a step-by-step install howto (without Docker) ?

Thank you very much!

Joe

**Comment by NickM-27** [comment]
We don't support bare metal installations without docker.

**Comment by sudo-joe** [comment]
Docker is not so nice in IPv6 environments and I don't know Docker at all :(

I would extremely appreciate it if you could you paste a minimal Docker-HowTo who is working for Frigate (instead of just pasting a link to docker.com or something like that).

Thank you very very much in advance:)

Joe
